### PR TITLE
create index.html

### PR DESCRIPTION
**Pull request by Kurzawskyy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,8 +4,47 @@
...
+                    <p class="points">Points: <b id="user-points">0</b></p>
+
+                    <div class="images">
+                        <img src="img/paper.png" alt="paper" height="170px" width="170px">
+                        <br>
+                        <img src="img/rock.jpg" alt="rock" height="170px" width="170px">
+                        <br> <br>
+                        <img src="img/scissors.png" alt="scissors" height="170px" width="170px">
+                    </div>
+                    <div id="choosen" class="choosen"></div>
```
Comment (Kurzawskyy) [inline index.html:35]: The selected picture will be inside this div (rock, paper or scissors)

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    margin-right: 25px;
+    font-size: 20px;
+}
+
+.images {
+    margin-left: 40px;
+    float: left;
+}
+
+.choosen {
```
Comment (Kurzawskyy) [inline styles.css:85]: This class defines div in which will be chosen picture. 
Border is optionally to see where is it and to position it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                <p id="round-number">Round: 1</p>
+            </div>
+
+            <div class="game">
+                <div class="player">
+                    <p id="user-name">User's name</p>
+                    <p class="points">Points: <b id="user-points">0</b></p>
+
+                    <div class="images">
+                        <img src="img/paper.png" alt="paper" height="170px" width="170px">
```
Comment (Lukasz-Semik) [inline index.html:28]: jpg has alwyas background - we should find `svg` or `png` - eventually - you are happy with current solution

File: index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                        <br>
+                        <img src="img/rock.jpg" alt="rock" height="170px" width="170px">
+                        <br> <br>
+                        <img src="img/scissors.png" alt="scissors" height="170px" width="170px">
+                    </div>
+                    <div id="choosen" class="choosen"></div>
```
Comment (Lukasz-Semik) [inline index.html:35]: could we add an empty line for better readabilty before this wrapper?

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    background: #5433FF;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #A5FECB, #20BDFF, #5433FF); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+    align-items: center;
+    justify-content: center;
+    display: flex;
+}
+
+.content-wrapper {
+    background-color: #ffffff;
```
Comment (Lukasz-Semik) [inline styles.css]: we can use shorter version `#fff`. 
Please use in all colors small or big letters (you can choose whatever you want)

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    padding: 15px;
+}
+
+.line {
+    float: left;
+    margin-top: 2%;
+    height: 91%;
+}
+
+#user-name {
```
Comment (Lukasz-Semik) [inline styles.css]: please, as I've showed you some time ago - don't use ids to styles. Only classes :)

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    color: #00cc00;
```
Comment (Lukasz-Semik) [inline styles.css]: shorthand: `#0c0`

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    font-size: 20px;
+}
+
+.images {
+    margin-left: 40px;
+    float: left;
+}
+
+.choosen {
+    float: left;
```
Comment (Lukasz-Semik) [inline styles.css:86]: using floating causes problems, let's discuss it on meeting :)

**Review by Lukasz-Semik — COMMENTED** [review]
good job 🎉  
Small comments

**Review by Lukasz-Semik — APPROVED** [review]

**Review by Lukasz-Semik — APPROVED** [review]

**Pull request merged**